edge/pkg/devicetwin/dtclient/sqlite_test.go

```diff
@@ -291,7 +356,6 @@ func TestQueryTwinAll(t *testing.T) {
 	for _, test := range cases {
 		t.Run(test.name, func(t *testing.T) {
 			querySeterMock.EXPECT().All(gomock.Any()).SetArg(0, *fakeTwin).Return(test.allReturnInt, test.allReturnErr).Times(1)
-			querySeterMock.EXPECT().Filter(gomock.Any(), gomock.Any()).Return(test.filterReturn).Times(1)
```

Comment:
Why remove this line?

Reply:
QueryTwinAll() function did not call the relevant Filter() function.  no need mock